README.md

```diff
@@ -1 +1,56 @@
-# Getting started with Cisco IOS-XE Smart licensing APIs
\ No newline at end of file
+# Getting Started with Cisco IOS-XE Smart licensing APIs
+
+Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
+
+For information about Smart Licensing Using Policy (SLP), see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
+
+## Use Case Description
+
+For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
```

Comment:
There are 5 APIs demonstrated in this script. 
They are as follows:
"Close-Reports" : To be used to close open usage reports.
"list" : To list all usage reports (open/closed?).
"all-reports" : list of all signed (open/closed?) reports along with DLC payload.
"days" : list of all signed reports from last N days
"import" : To import report ACK, new policy, or Authorization code into the product instance.

The script does not interact with CSSM (Cisco Smart Software Manager) directly.
It is assumed that the CSSM interactions are done manually in offline manner by file upload/download.

Reply:
But we've provided this same info in the About the APIs section - shall I provide a reference to this section? 

https://developer.cisco.com/docs/ios-xe/smart-licensing/#!about-the-apis/api-endpoint-and-description